site/en/blog/extension-safety-hub/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Bringing Safety Check to the chrome://extensions page"
+description: "Starting in Chrome 117, Chrome will proactively flag to users when an extension they have installed is no longer in the Chrome Web Store."  
+layout: "layouts/blog-post.njk"
+authors:
+  - oliverdunk
+date: 2023-08-16
+tags:
+  - extensions-news
+---
+
+Starting in Chrome 117, Chrome will proactively flag to users when an extension they have installed is no longer in the Chrome Web Store. This is limited to three specific cases:
+
+- The extension has been unpublished by the developer.
```

Comment:
```suggestion
- The extension was taken down by the developer.
```

Reply:
Do you feel strongly on this? While they mean the same in general they have very different meanings to developers - "taken down" usually has negative connotations and is associated with something we do for malicious activity, whereas "unpublished" just means a developer deciding to unlist their extension. I used the two in different places intentionally.